python/triton/language/core.py

```diff
@@ -419,6 +423,23 @@ def scalar(self):
         return self
 
 
+class const_pointer_type(pointer_type):
```

Comment:
@ptillet This may be misleading too, should I rename it to pointer_to_const_type? There is a fine line between being too verbose and misleading.

Reply:
yep, I think this would be better